frontend/components/dynamic-badge-maker.js

```diff
@@ -42,6 +42,7 @@ export default class DynamicBadgeMaker extends React.Component {
           onChange={event => this.setState({ datatype: event.target.value })}>
           <option value="" disabled>data type</option>
           <option value="json">json</option>
+          <option value="yaml">yaml</option>
```

Comment:
Can we sort them in an alphabetical order (json, xml, yaml)?

Reply:
See other comment about order.